Makefile

```diff
@@ -1,5 +1,52 @@
+all: test check-coverage lint tidy ## Tests, lints and checks coverage
+
+clean:  ## Remove generated files
+
+.PHONY: all clean
+
+# -- Lint ----------------------------------------------------------------------
+lint: ## Lint Go Source Code
+	golangci-lint run
+
+tidy:
+	go mod tidy
+
+.PHONY: lint tidy clean
```

Comment:
no clean target here?

Reply:
Thanks, fixed in 728d9a010242a1d6784cb6665c4f590129a3b23a